plugin-server/src/cdp/templates/_transformations/bot-detection/bot-detection.template.ts

```diff
@@ -104,10 +104,215 @@ fun isBot(ua) {
 
 // If bot is detected, return null to filter out the event
 // Otherwise return the original event
-if (isBot(user_agent)) {
+if (isBotUa(user_agent)) {
     return null
 }
 
+let known_bot_ip_prefixes := [
```

Comment:
This is yet another big memory footprint for virtually static content... @meikelmosby just a reminder of something that would be cool to think about is if we can abstract this somehow...

Maybe it would make sense to have a shared function called like "getCommonBotIpPrefixes()". Would be a little specific but would for sure help with having a tonne of duplicate code everywhere 🤔